App/UI/UploadDataAutonomous/Cells/TextFieldCun.swift

```diff
@@ -164,6 +169,15 @@ extension TextFieldCun: UITextFieldDelegate {
         let protectedRange = NSRange(location: 0, length: 4)
         let intersection = NSIntersectionRange(protectedRange, range)
 
+        if range.location < 4 || textField.text?.count ?? 0 > 13 {
+            if string == "" && range.location > 3{
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/UI/UploadDataAutonomous/Cells/TextFieldCun.swift;
  Line: 173;
-->

- :warning: swiftlint (0.39.2): Prefer checking `isEmpty` over comparing `string` to an empty string literal. (empty_string)